### ISSUE TITLE
PostCSS warning about 'from' option

### ISSUE THREAD

**Issue by 11bits** [kentaroi/eleventy-sass#11]
When I run this plugin, I get the following warning:

"Without `from` option PostCSS could generate wrong source map and will not find Browserslist config. Set it to CSS file path or to `undefined` to prevent this warning."

This is my Eleventy setup:

```
const autoprefixer = require("autoprefixer");
const eleventySass = require("eleventy-sass");
const postcss = require("postcss");

module.exports = function (config) {
  config.addPlugin(eleventySass, {
    postcss: postcss([autoprefixer]),
    sass: {
      style: "compressed",
      sourceMap: false,
    },
  });
...
```

Is it possible to add that option to the Eleventy config file?

**Comment by kentaroi** [comment]
I've not noticed the warning.
Thank you very much for filing the issue.

I'll publish new major version of the plugin soon, and I'll include the fix to address the issue and/or the feature that allows users to customize options for postcss.

**Comment by kentaroi** [comment]
I just published v2.0.0, which includes the fix.

Since this is a major release, please update the plugin like this:
```bash
npm install eleventy-sass@2
```
